src/extensions/husky-setup-extension.test.js

```diff
@@ -1,61 +1,307 @@
-const extend = require('./husky-setup-extension');
-const { createToolboxMock, createExtensionInput } = require('../utils/test/mocks');
+const extend = require('./husky-setup-extension')
+const {
+  createToolboxMock,
+  createExtensionInput,
+} = require('../utils/test/mocks')
 
 describe('husky-setup-extension', () => {
   it('should be defined', () => {
-    expect(extend).toBeDefined();
-  });
+    expect(extend).toBeDefined()
+  })
 
   it('should set setupHusky on toolbox', () => {
-    const toolbox = {};
-    extend(toolbox);
-    expect(toolbox.setupHusky).toBeDefined();
-  });
+    const toolbox = {}
+    extend(toolbox)
+    expect(toolbox.setupHusky).toBeDefined()
+  })
 
   describe('setupHusky', () => {
-    const toolbox = createToolboxMock();
+    const toolbox = createToolboxMock()
 
     beforeAll(() => {
-      extend(toolbox);
-    });
+      extend(toolbox)
+    })
 
     it('should return syncOperations and asyncOperations when the extension is called', () => {
-      const ops = toolbox.setupHusky(createExtensionInput());
-      expect(ops.syncOperations).toBeDefined();
-      expect(ops.asyncOperations).toBeDefined();
-    });
+      const ops = toolbox.setupHusky(createExtensionInput())
+      expect(ops.syncOperations).toBeDefined()
+      expect(ops.asyncOperations).toBeDefined()
+    })
 
     describe('syncOperations', () => {
-      const toolbox = createToolboxMock();
-      const input = createExtensionInput();
+      const toolbox = createToolboxMock()
```

Comment:
I see a lot of these shadowing declarations which I personally don't like, as here they are redundant and a repeating block of code. We can get rid of them if we put the `toolbox` and `input` declaration in the top `setupHusky` `describe` block (now there we have only the `toolbox` one) and change all the `beforeAll` to `beforeEach` (not needed for every block, but it would be more consistent like that)